forge-gui/res/cardsfolder/upcoming/mace_of_disruption.txt

```diff
@@ -0,0 +1,9 @@
+Name:Mace of Disruption
+ManaCost:1 W
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddToughness$ 2 | AddKeyword$ Protection from Demons & Protection from Devils | Description$ Equipped creature gets +0/+2 and has protection from Demons and from Devils.
+T:Mode$ Attacks | ValidCard$ Creature.EquippedBy | Execute$ TrigEffect | TriggerDescription$ Whenever equipped creature attacks, it perpetually gets +1/+0 if it has the same name as another creature you control or a creature card in your graveyard.
+SVar:TrigEffect:DB$ Effect | ConditionDefined$ Equipped | ConditionPresent$ Creature.sharesNameWith OtherYourBattlefield,Creature.sharesNameWith YourGraveyard | RememberObjects$ Equipped | Name$ Mace of Disruption's Perpetual Effect | StaticAbilities$ PerpetualP1P0
```

Comment:
Not a huge fan of hard-coding restrictions like that
What you think of doing this more like on _Ixalan's Binding_?
Just as suggestion though

Reply:
The mixed zones thing ended up pushing me out of the in-line ConditionPresent – but at least we didn't hardwire another thing

Reply:
I misjudged the need for RepeatEach though
but probably good enough for Alchemy